cmd/device-worker/main.go

```diff
@@ -78,7 +78,7 @@ func main() {
 		log.Fatal(err)
 	}
 	if !r.GetRegistered() {
-		log.Fatal("handler registration failed")
+		log.Fatal("yggdrasil registration failed")
```

Comment:
Why do we need `yggdrasil` in this log entry? Maybe we could use `Registration failed`?

Reply:
this is registration from device-worker to yggdrasil server.